src/Microsoft.DocAsCode.Build.TableOfContents/TocResolver.cs

```diff
@@ -45,7 +45,9 @@ private TocItemInfo ResolveItemCore(TocItemInfo wrapper, Stack<FileAndType> stac
             var file = wrapper.File;
             if (stack.Contains(file))
             {
-                throw new DocumentException($"Circular reference to {file.FullPath} is found in {stack.Peek().FullPath}");
+                var message = $"Circular reference to {file.FullPath} is found in {stack.Peek().FullPath}";
+                Logger.LogError(message, code: ErrorCodes.Build.CircularReferenceFound);
```

Comment:
We can group these together: `ErrorCodes.Toc.XXX`.

Reply:
Does this log code - "CircularReferenceFound" sound good?

Reply:
It can be `ErrorCodes.Toc.CircularTocInclusion`.